kpi/tests/api/test_api_environment.py

```diff
@@ -100,15 +104,7 @@ def setUp(self):
                 )
             ),
             'mfa_code_length': settings.TRENCH_AUTH['CODE_LENGTH'],
-            'stripe_public_key': (
-                str(
-                    APIKey.objects.get(
-                        type='publishable', livemode=settings.STRIPE_LIVE_MODE
-                    ).secret
-                )
-                if settings.STRIPE_ENABLED
-                else None
-            ),
+            'stripe_public_key': None,
```

Comment:
Let's leave a comment here that `stripe_public_key` can be overridden below when Stripe is enabled.